bite-size-notes/week-6-c-state-estimation-intro.md

```diff
@@ -0,0 +1,193 @@
+
+## Preliminaries
+
+First, we'll begin with a few definitions of relevant concepts used throughout the rest of the notes.
+
+### Relevant Definitions:
+
+- **Definition 1:**
+
+    A system is considered observable if, for any given time $T > 0$, it is possible to determine the system's state at that time $x(T) \in \mathbb{R}^n$ through measuring control input $u(t)$ and output $y(t)$ where $t \in [0,T]$
+
+    A pair of dynamics $(A,C)$ is observable if the observability matrix $O$ has $rank(O) = n = dim(A)$. The observability matrix is as follows:
+
+
+$$ \ O =    \begin{bmatrix}
+                C\\ CA \\\vdots \\ CA^{n-1}
+            \end{bmatrix}\ 
+$$
+
+- **Definition 2:**
+
+    The pair $(A, C)$ is detectable if all the unobservable modes are stable.
+
+- **Definition 3:**
+
+    A linear system is reachable if for any $x_o, x_f \in \mathbb{R}^n$, there exists a $T>0$ and $u(t): [0, T] \rightarrow \mathbb{R}$ such that the corresponding solution satisfies $x(0) = x_o$ and $x(T) = x_f \in \mathbb{R}^n$
+
+- **Definition 4:**
+
+    A system (A,B) is reachable if and only if $rank(R) = n$ where $x \in \mathbb{R}^n$ and
+
+    $$
+    R = \begin{pmatrix}
+    B & AB & \dots & A^{n-1}B
+    \end{pmatrix}
+    $$
+
+### Problem Definition:
+
+Given a linear, time invariant system ignoring noise, we have state and measurement output
+
+$$ 
+\dot{x}(t) = Ax(t) + Bu(t) \\
+y(t) = Cx(t) + Du(t)
+$$
+
+From this, we can estimate the state $\hat{x}(t)$ using measurements using control inputs and measured outputs. To do this we want error $\tilde{x}(t) = x(t) - \hat{x}(t)$ to converge to 0. In other words,
+
+$$
+\lim_{t\rightarrow \infty}(\tilde{x}) = 0
+$$
+
+Therefore, this now becomes a clear control problem as we're effectively trying to minimize our error.
+
+## Main Body
+
+### The Luenberger Observer
+
+The Luenberger Observer is a deterministic state estimator. First, we create create estimator copies of the original system.
+
+$$ 
+\dot{\hat{x}}(t) = Ax(t) + Bu(t) \\
+\hat{y}(t) = Cx(t) + Du(t)
+$$
+
+Now expanding our error dynamics:
+
+$$
+\dot{\tilde{x}}(t) = \dot{x}(t) - \dot{\hat{x}}(t) = Ax(t) + Bu(t) - A\hat{x}(t) - Bu(t) = A\hat{e}(t)
+$$
+
+Now we can clearly see two situations arise from the following. Either the matrix A has all eigenvalues in the left half-plane which will allow $\hat{e}(t)$ to converge to zero. While this may converge to 0, it may be impractically slow and doesn't utilize the measurement as a feedback system. Additionally, we cannot guarantee the matrix A to be stable. Adding feedback only requires adding matrix $L$ to the estimator.
+
+$$ 
+\dot{\hat{x}}(t) = Ax(t) + Bu(t) + L(y(t) - \hat{y}(t))
+$$
+
```

Comment:
Could you make your expanding of our error dynamics with new estimated state dynamics more clearly? Maybe something incorrect with your editing your formula

Reply:
Fixed!